account_fiscal_year/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Camptocamp SA
+# Copyright 2018 Lorenzo Battistini <https://github.com/eLBati>
+# Copyright 2020 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Account Fiscal Year",
+    "summary": "Create Account Fiscal Year",
+    "version": "17.0.1.0.0",
+    "development_status": "Production/Stable",
+    "category": "Accounting",
+    "website": "https://github.com/OCA/account-financial-tools"
+    "14.0/account_fiscal_year",
```

Comment:
`17.0` or `https://github.com/OCA/account-financial-tools` even better

Reply:
changed to https://github.com/OCA/account-financial-tools only.